src/status_im/utils/test.cljs

```diff
@@ -15,17 +15,21 @@
 
 (def test-dir (.mkdtempSync fs test-dir-prefix))
 
+(def initialized? (atom false))
+
 (defn signal-received-callback
   [a]
   (re-frame/dispatch [:signals/signal-received a]))
 
 ;; We poll for signals, could not get callback working
 (defn init!
   []
-  (.setSignalEventCallback native-status)
-  (js/setInterval (fn []
-                    (.pollSignal native-status signal-received-callback)
-                    100)))
+  (when-not @initialized?
```

Comment:
FYI: Here we make `init!` idempotent, so we can safely and automatically call it for every test using the macro `with-app-initialized`. This is what allows each namespace to run independently and in any order.